whois-query/src/test/java/net/ripe/db/whois/query/integration/WhoisConnectionTestIntegration.java

```diff
@@ -39,56 +25,51 @@
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ContextConfiguration;
 
-import java.net.InetAddress;
 import java.nio.channels.ClosedChannelException;
-import java.nio.charset.StandardCharsets;
 
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.assertEquals;
-import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.anyInt;
 import static org.mockito.Mockito.doAnswer;
 import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.when;
 
+@Ignore("TODO [DA] I suggest we remove this test. For one, the changes in the API makes mocking cumbersome. But the main reason is that the utility of the test can be argued." +
```

Comment:
@eshryane @sbusk any objections to the suggestion to remove this file?

Reply:
Sounds ok to me

Reply:
Removed in commit 2940ce1